src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -121,6 +121,71 @@
                     </Setter.Value>
                 </Setter>
             </Style>
+            <Style x:Key="UnSaveButtonStyle"
+                   TargetType="{x:Type Button}"
+                   BasedOn="{StaticResource CtaButtonStyle}">
+                <Setter Property="Height" Value="35"></Setter>
+                <Setter Property="Width" Value="120"></Setter>
+                <Setter Property="Margin" Value="15 5 20 5"></Setter>
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate>
+                            <Border x:Name="container"
+                                    Background="Transparent"
+                                    BorderBrush="#B8ABA1"
+                                    CornerRadius="2"
+                                    BorderThickness="1">
+                                <Grid x:Name="inner"
+                                      Background="Transparent">
+                                    <TextBlock x:Name="text"
+                                               HorizontalAlignment="Center"
+                                               VerticalAlignment="Center"
+                                               FontSize="14px"
+                                               Foreground="#3C3C3C"
```

Comment:
Use the color variables from Dynamo brushes files, wherever possible.

Reply:
Resolved